Atomic/client.py

```diff
@@ -51,7 +56,10 @@ def __getattribute__(self, name):
         if name == "_dockerclient" or name == "_client":
             return object.__getattribute__(self, name)
         obj = self._dockerclient
-        attr = docker.AutoVersionClient.__getattribute__(obj, name)
+        try:
```

Comment:
Could you reverse these.  Since going forward APIClinet will be default lets make that the primary and Client be the fall back.